monkey/monkey_island/cc/deployment.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
why do we need this file?

Reply:
As discussed in planning, this is the default file.